test/AffInterface.purs

```diff
@@ -40,6 +52,46 @@ suite = do
       $ testFromOgmiosAddress testnet_addr1
     test "Ogmios Address to Address & back non-Testnet"
       $ testFromOgmiosAddress addr1
+  group "Ogmios datum cache" do
+    test "Can process GetDatumByHash" do
+      testOgmiosDatumCacheGetDatumByHash
+    test "Can process GetDatumsByHashes" do
+      testOgmiosDatumCacheGetDatumsByHashes
+    test "Fetcher works" do
+      testOgmiosDatumCacheFetcher
+
+testOgmiosDatumCacheGetDatumByHash :: Aff Unit
+testOgmiosDatumCacheGetDatumByHash =
+  traceQueryConfig >>= flip runQueryM do
+    -- Use this to trigger block fetching in order to actually get the datum:
```

Comment:
See my larger comment about the flake. We can just automate this part, I think this will be error-prone (IME a lot of users have found the explicit block fetching unclear)

Reply:
Do you want to remove the start/stop calls from our code?

Reply:
Hmmm, good question. I suggested removing these because we should use the automated fetching that ODC now supports. I think users (and frankly even CTL developers who haven't had to deal with ODC yet) won't know to run the block fetcher. Maybe mention the autofetching in the comment?

Also, we could move `testOgmiosDatumCacheFetcher` to before any other tests, then the block fetcher will have been stopped even if is automatically run from the runtime config

At any rate, I'll leave the ultimate decision up to you